installing/install_config/customizations.adoc

```diff
@@ -29,46 +29,55 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |Configure the Kubernetes API server.
 
 |Authentication
-|You can configure the identity provider for your cluster.
+|You can configure the
+xref:../../authentication/understanding-identity-provider.adoc#understanding-identity-provider[identity provider]
+for your cluster.
 
 |Build
 |Configure build settings.
 
 |ClusterVersion
-|Configure the desired version of the cluster and how updates are handled.
+|In {product-title} {product-version}, you must not customize the ClusterVersion
+resource for production clusters. Instead, follow the process to
+xref:../../updating/updating-cluster.adoc#updating-cluster[update a cluster].
 
 |Console
 |Configure the web console.
```

Comment:
@spadgett are there docs that can be linked here, or is someone working on them?

Reply:
@spadgett said that you can only configure the redirect URL for the console. I asked for an example of the YAML object to modify so that we can add this.

Reply:
This is here: https://github.com/openshift/openshift-docs/pull/15097